packages/connect/src/core/AbstractMethod.ts

```diff
@@ -333,6 +336,24 @@ export abstract class AbstractMethod<Name extends CallMethodPayload['method'], P
         };
     }
 
+    checkDeviceCapability() {
+        const deviceHasAllRequiredCapabilities = (this.requiredDeviceCapabilities || []).every(
+            capability => this.device.features.capabilities.includes(capability),
+        );
+        if (!deviceHasAllRequiredCapabilities) {
+            if (this.device.firmwareType === 'bitcoin-only') {
+                throw ERRORS.TypedError(
+                    'Device_MissingCapabilityBtcOnly',
+                    `Trezor has Bitcoin-only firmware installed, which does not support the ${this.info} method. Please install Universal firmware through Trezor Suite.`,
```

Comment:
this.info may sometimes yield quite variable results 🤔 just thinking whether it is a good idea to intertwine it together here

Reply:
Yeah we could just write "does not support this method" or maybe "does not support this operation", since method may sound strange for less technical people